docs/framework/data/adonet/connection-string-syntax.md

```diff
@@ -72,7 +72,7 @@ When you connect to Azure SQL Database or to Azure SQL Data Warehouse and provid
 ### Connect to a named instance of SQL Server
 To connect to a named instance of SQL Server, use the *server name\instance name* syntax.  
   
-```  
+```csharp  
 Data Source=MySqlServer\MSSQL1;"  
 ```  
```

Comment:
That's not related to the language identifier, I think there is a missing double quote.

Reply:
Good catch. I made the fix.